mesos-rxjava-recordio/pom.xml

```diff
@@ -37,4 +37,21 @@
         </dependency>
     </dependencies>
 
+    <build>
+        <plugins>
+            <plugin>
```

Comment:
It might not be terrible to move to to the top level so that all modules will be able to publish their test-jars, I could see this being useful in the future.